content/modules/rest/pages/entities-api/update-entities.adoc

```diff
@@ -0,0 +1,404 @@
+      "name": "great"
+    }
+  ]
+}
+----
+
+In this example you would like to change the tags of the Product in the following ways:
+
+* `great` should be removed from the tags.
+* `amazing` should be added to the tags.
+* The tag `shiny` should be kept.
+
+[source, http request]
+.Update Tag Associations Request
+----
+PUT http://localhost:8080/rest
+            /entities
+            /rstex11_Product
+            /e1d586b4-aefb-2ee7-3b91-b07357b178ea
+            ?responseFetchPlan=product-with-tags
+
+{
+  "name": "123",
+  "price": 99.95,
+  "tags": [
+    {
+      "id": "333f3a20-c47b-4bc9-ba34-a72d2d815695" // <1>
+    },
+    {
+      "id": "d6ab132e-a0bd-a624-c6ad-cc544e83c584" // <2>
+    }
+  ]
+}
+----
+<1> The ID of the product tag `shiny` is part of the request in order to keep the association.
+<2> The ID of the product tag `amazing` is newly added to the association.
+
+NOTE: As the ID of the product tag `great (c4c028f0-fec1-7512-83cd-c17537d1f502)` is not part of the request anymore, it will be removed from the association.
+
+[source,json]
+.Response: 200 - OK
+----
+{
+  "_entityName": "rstex11_Product",
+  "_instanceName": "Outback Power Remote Power System",
+  "id": "e1d586b4-aefb-2ee7-3b91-b07357b178ea",
+  "createdDate": "2021-03-05",
+  "createdBy": "admin",
+  "lastModifiedDate": "2021-03-05",
+  "price": 99.95,
+  "lastModifiedBy": "admin",
+  "name": "Outback Power Remote Power System",
+  "version": 2,
+  "tags": [
+    {
+      "_entityName": "rstex11_ProductTag",
+      "_instanceName": "shiny",
+      "id": "333f3a20-c47b-4bc9-ba34-a72d2d815695",
+      "name": "shiny" // <1>
+    },
+    {
+      "_entityName": "rstex11_ProductTag",
+      "_instanceName": "amazing",
+      "id": "d6ab132e-a0bd-a624-c6ad-cc544e83c584",
+      "name": "amazing" // <2>
+    }
+  ]
+}
+----
+<1> The `shiny` reference is still there, as it was part of the request
+<2> The `amazing` reference has been added, whereas the tag `great` is not part of the association anymore.
+
+[CAUTION]
+.Remove `*:1` Entity References
+====
+In order to remove a reference for `N:1` or `1:1` associations, you need to send in `null` as a value. The absence of the attribute in the request will _not_ lead that the reference is removed, because of the <<Partial Updates>> feature. In this case the attribute would just be ignored and not changed as part of the request.
+====
+
+=== Composition Attributes
+
+In case you want to update a Composition attribute, it is possible to directly update the content of the child entity as part of the update request for the parent entity. This is true for `1:1` as well as `1:N` compositions.
+
+IMPORTANT: the Update Entity endpoint will _replace_ the existing composition collection with what is part of the request. For every entity reference that is not part of the request the reference will be removed. Further: the entity that has been referenced before _will be deleted_ from the application as well.
```

Comment:
See my comments on admnition style above.
And I would replace "Further:" → "Moreover,"